### PR TITLE
add new scaffolder action to ensure a gitlab group exists

### PR DESCRIPTION
**Pull request by Andy2003**
With this action it can be ensured, that a Gitlab group exists.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by Andy2003** [comment]
Rlease do not close this PR! I wait for the Review!

**Comment by Andy2003** [comment]
Still wait for PR review!

@Rugvip can you please reassign this PR, in case you have no time to review it.

**Comment by manobi** [comment]
This would make my automations so much simples.
Looking forward to see that merged.

File: plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabProjectAccessTokenAction.ts
Comment (Rugvip) [inline plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabProjectAccessTokenAction.ts:43]: Should be a `number` here right?

File: plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabGroupEnsureExistsAction.ts
Comment (Rugvip) [inline plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabGroupEnsureExistsAction.ts:69]: Perhaps add a comment here since this is quite unexpected, just explaining that the type is actually wildly wrong

**Review by Rugvip — CHANGES_REQUESTED** [review]
Thank you! 👍 🎉 

Just some minor stuff, sorry it took at while to review

**Comment by Andy2003** [comment]
@Rugvip any updates on this?

**Comment by Andy2003** [comment]
Still wait for PR approval!

**Comment by Andy2003** [comment]
Still wait for PR approval!

**Comment by Andy2003** [comment]
Still wait for PR approval!

@Rugvip can you please reassign this PR, in case you have no time to review it.

**Review by Rugvip — APPROVED** [review]
All good, thank you! 👍 🎉 

We're all busy, in particular the last few weeks. Working on scaling things though 😁

**Pull request merged**
